_tags

```diff
@@ -10,7 +10,10 @@ true: debug
 <src/*.ml>: for-pack(Key_parsers)
 <src/asn1.*>: package(ppx_deriving.std), package(ppx_deriving_yojson)
 <src/ltpa.*>: package(ppx_deriving.ord), package(ppx_deriving_yojson)
+<src/cvc.*>: package(ppx_deriving.ord), package(ppx_deriving_yojson), package(ppx_deriving.eq), package(ppx_deriving.show)
```

Comment:
That looks like premature optimization, `true: package(ppx_deriving.std)` would work as well + be more maintainable. Let's do this in a separate PR.